src/analytics.js

```diff
@@ -35,6 +36,12 @@ if ( ! Config.PinpointId || ! Config.CognitoId ) {
 	/* eslint-enable quotes */
 }
 
+/**
+ * Get consent types.
+ */
+let hasAnonConsent = Consent.CookiePrefix && document.cookie.match( `${ Consent.CookiePrefix }_statistics-anonymous=allow` );
```

Comment:
Is there no JS API in our consent API to check if someone has already consented?

Reply:
There is yeah but this script is loaded super early hence this approach.

Reply:
Ahh got you! Might be good to add a some notes about that in the doc comment

Reply:
Added, thanks @joehoyle